examples/min-max-vector/src/main.rs

```diff
@@ -7,6 +7,14 @@ fn main() {
     let max = numbers.iter().max().unwrap();
     println!("max: {}", max);
 
+    let (min, max) = numbers
```

Comment:
I would normally put this at the bottom of the function. but on line 22 `numbers` moves, so this example comes before.
I can move this example to the bottom and clone `numbers` if this is super critical